presto-orc/pom.xml

```diff
@@ -16,6 +16,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```

Comment:
put the dependency in the parent pom and not have the version here, similar to other dependencies

Reply:
+1

Reply:
Fixed this in a couple other places too